File: src/main/java/com/musinsa/subject/repository/BrandRepository.java
package com.musinsa.subject.repository;

import com.musinsa.subject.entity.Brand;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface BrandRepository extends JpaRepository<Brand, Long> {

    List<Brand> findAllByOrderByBrandId();

}

File: src/main/kotlin/com/musinsa/subject/repository/BrandRepository.kt
package com.musinsa.subject.repository

import com.musinsa.subject.entity.Brand
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface BrandRepository : JpaRepository<Brand, Long> {
    fun findAllByOrderByBrandId(): List<Brand>
}
